packages/backend/src/models/AnalyticsModel.ts

```diff
@@ -38,12 +37,36 @@ export class AnalyticsModel {
         this.database = dependencies.database;
     }
 
-    async countChartViews(chartUuid: string): Promise<number> {
-        const [{ count }] = await this.database(AnalyticsChartViewsTableName)
-            .count('chart_uuid')
-            .where('chart_uuid', chartUuid);
+    async getChartViewStats(chartUuid: string): Promise<ViewStatistics> {
+        const transaction = Sentry.getCurrentHub()
+            ?.getScope()
+            ?.getTransaction();
+        const span = transaction?.startChild({
+            op: 'AnalyticsModel.getChartStats',
+            description: 'Gets a single chart statistics',
+        });
 
-        return Number(count);
+        try {
+            const stats = await this.database(AnalyticsChartViewsTableName)
+                .count({ views: '*' })
+                .min({
+                    first_viewed_at: 'timestamp',
+                })
+                .select('chart_uuid')
+                .groupBy('chart_uuid')
```

Comment:
You don't need the `.select()` or `.groupBy` here. You can just have the two aggregations (count, min)